haiku-apps/armyknife/armyknife-5.1.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="The all-in-one metadata editor for audio and MPEG files"
+DESCRIPTION="ArmyKnife is an application that lets you edit the metadata of \
+audio and MPEG files, both Haiku file attributes and in-file ID3 tags. You can \
+copy data to and from ID3 tags and attributes, and rename files according to \
+attributes or fill attributes according to file name.
+Now also with support for cover art!"
+HOMEPAGE="https://github.com/HaikuArchives/ArmyKnife"
+COPYRIGHT="2000-2001 Jason Burgess"
```

Comment:
Maybe you should add yourself here since you did some major improvements to the app?